### ISSUE TITLE
Error Rate (Validation + Test)

### ISSUE THREAD

**Issue by gyan2976** [kartikgill/Easter2#10]
Hi. I have used your checkpoint hdf5, change the path according to config.py and done the changes ('tensorflow':tf, 'K':K), but I am not able to achieve the accuracy that you have got. Can you please look at this once.

CER on Validation Set
![Validation](https://user-images.githubusercontent.com/103508999/201835086-9eb44c62-42d7-4fae-9d56-bcf05b14bd45.png)

CER or Test Set
![Test](https://user-images.githubusercontent.com/103508999/201835126-0215337a-82b5-4589-a656-a28e2e9c271d.png)

**Comment by kartikgill** [comment]
Hey @grkumar123 ,

Sorry, I couldn't respond to your query earlier...had been really busy with other work. If you are still facing the issue - Kindly provide your full code...so that we can better understand the issue. 

Again, thanks for trying out Easter.
